source/_sass/page.scss

```diff
@@ -1,333 +1,330 @@
 }
 // When JavaScript is enabled, we hide the menu
 .js .nav ul {
-	max-height: 0;
-	overflow: hidden;
-	@media #{$small} {
-		max-height: 30em;
-	}
+  max-height: 0;
+  overflow: hidden;
+  @media #{$small} {
+    max-height: 30em;
+  }
 }
 // Displaying the menu when the user has clicked on the button
 .js .nav .active + ul {
-	max-height: 30em;
-	overflow: hidden;
-	-webkit-transition: max-height .4s;
-	-moz-transition: max-height .4s;
-	-o-transition: max-height .4s;
-	-ms-transition: max-height .4s;
-	transition: max-height .4s;
+  max-height: 30em;
+  overflow: hidden;
+  -webkit-transition: max-height .4s;
+  -moz-transition: max-height .4s;
+  -o-transition: max-height .4s;
+  -ms-transition: max-height .4s;
+  transition: max-height .4s;
 }
 #main {
-	counter-reset: captions;
-	@include container;
-	@include clearfix;
-	clear: both;
-	margin-top: 2em;
-	h1 {
-		margin-top: 0;
-	}
-	.post,
-	.page {
-		@include container;
-		@include grid(12,10);
-		@include prefix(12,1);
-		@include suffix(12,1);
-		margin-bottom: 2em;
-		@media #{$small} {
-			@include grid(12,6);
-			@include prefix(12,0.5);
-			@include suffix(12,0.5);
-		}
-		@media #{$x-large} {
-			@include grid(12,4.5);
-			@include prefix(12,2);
-		}
-	}
+  counter-reset: captions;
+  @include container;
+  @include clearfix;
+  clear: both;
+  margin-top: 2em;
+  h1 {
+    margin-top: 0;
+  }
+  .post,
+  .page {
+    @include container;
+    @include grid(12,10);
+    @include prefix(12,1);
+    @include suffix(12,1);
+    margin-bottom: 2em;
+    @media #{$small} {
+      @include grid(12,6);
+      @include prefix(12,0.5);
+      @include suffix(12,0.5);
+    }
+    @media #{$x-large} {
+      @include grid(12,4.5);
+      @include prefix(12,2);
+    }
+  }
 }
 #index {
-	@include container;
-	@include grid(12,10);
-	@include prefix(12,1);
-	@include suffix(12,1);
-	margin-bottom: 2em;
-	@media #{$small} {
-		@include grid(12,6);
-		@include prefix(12,0.5);
-		@include suffix(12,0.5);
-	}
-	@media #{$x-large} {
-		@include grid(12,4.5);
-		@include prefix(12,2);
-	}
-	h3 {
-		margin: 0;
-		padding-bottom: .5em;
-		@include font-rem(28);
-		border-bottom: 1px solid lighten($black,70);
-	}
-	article {
-		h2 {
-			margin-bottom: 4px;
-			@include font-rem(20);
-			&.link-post {
-				margin-bottom: 0px + $doc-line-height;
-				margin-bottom: 0rem + ($doc-line-height / $doc-font-size);
-			}
-		}
-		p {
-			@include font-rem(14);
-		}
-		p+p {
-			text-indent: 0;
-		}
-	}
+  @include container;
+  @include grid(12,10);
+  @include prefix(12,1);
+  @include suffix(12,1);
+  margin-bottom: 2em;
+  @media #{$small} {
+    @include grid(12,6);
+    @include prefix(12,0.5);
+    @include suffix(12,0.5);
+  }
+  @media #{$x-large} {
+    @include grid(12,4.5);
+    @include prefix(12,2);
+  }
+  h3 {
+    margin: 0;
+    padding-bottom: .5em;
+    @include font-rem(28);
+    border-bottom: 1px solid lighten($black,70);
+  }
+  article {
+    h2 {
+      margin-bottom: 4px;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets